hermes/crates/cardano-chain-follower/src/multi_era_block_data.rs

```diff
@@ -33,6 +33,7 @@ struct SelfReferencedMultiEraBlock {
 #[derive(Debug)]
 pub struct MultiEraBlockInner {
     /// What blockchain was the block produced on.
+    #[allow(dead_code)]
```

Comment:
are you sure that we need this `#[allow(dead_code)]` here ?

Reply:
Yes. This field prevents me from running `build`, so I need to add `#[allow(dead_code)]` to make it run.